template/drush/README.md

```diff
@@ -1,3 +1,13 @@
+# Drush configuration and aliases
+
 This directory is intended to contain drush configuration that is not site or environment specific.
 
 Site specific drush configuration lives in `sites/[site-name]`.
+
+## Site aliases
+
+You should add the Drush aliases for any remote environments (such as Acquia Cloud) to the `site-aliases` directory. This allows developers to access remote environments using simple aliases such as `drush @mysite.dev uli`. Note that if you are using Acquia Cloud, developers can also download these aliases manually from their Insight account, but providing them with the project makes everyone’s life a little easier.
```

Comment:
I did add this to bolt,
https://github.com/acquia/bolt/blob/8.x/template/drush/site-aliases/example.local.aliases.drushrc.php